mlrun/datastore/base.py

```diff
@@ -419,14 +421,34 @@ def as_df(
         :param df_module: optional, py module used to create the DataFrame (e.g. pd, dd, cudf, ..)
         :param format:    file format, if not specified it will be deducted from the suffix
         """
-        return self._store.as_df(
+        if "time_column" in kwargs or "time_field" in kwargs:
+            kwargs["time_column"] = kwargs.get("time_column") or kwargs.pop(
+                "time_field", None
+            )
```

Comment:
I think we should change these into proper parameters. They don't really belong in kwargs.